src/Voron/Data/Fixed/FixedSizeTree.cs

```diff
@@ -57,6 +58,8 @@ public static ushort GetValueSize(LowLevelTransaction tx, Tree parent, Slice tre
 
         public void RepurposeInstance(Slice treeName, bool clone)
         {
+            _addScope = new DirectAddScope(this);
```

Comment:
Can we avoid this? How about allocate it once in ctor, and then require that if it get here, the previous call has been disposed?

This method is called a _lot_ and the whole reason it exists is that we want to avoid allocations.